diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$("li")
+$('li:first-child') *i'm pretty sure this will only return the first li element, but I couldn't find another in option in time
```

Comment:
Yup, this is correct. Here are some other options: `$('body').find('li');` and `$('body').children('ol').children('li')`